src/NUnitFramework/framework/Internal/AttributeProviderWrapper.cs

```diff
@@ -23,12 +23,11 @@
 
 using System;
 using System.Collections.Generic;
-using System.Linq;
 using System.Reflection;
 
 namespace NUnit.Framework.Internal
 {
-    internal class AttributeProviderWrapper<T> : ICustomAttributeProvider
+    internal sealed class AttributeProviderWrapper<T> : ICustomAttributeProvider
```

Comment:
Thanks for this 😄

Reply:
Ideally all private/internal things should be sealed, it allows CLR some more room to optimize. This is what the coreclr/runtime did, made everything sealed they could. Might be a micro-optimization, but can be important to things like comparers/equality checkers.